scripts/benchmark_compare.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
```

Comment:
You'll need to add this file to scripts/include.am.

Reply:
Yup. Already done locally. Didn't want to trigger another build